src/main/kotlin/org/rust/ide/lineMarkers/RsGithubIssueLineMarkerProvider.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.lineMarkers
+
+import com.intellij.codeInsight.daemon.LineMarkerInfo
+import com.intellij.codeInsight.daemon.LineMarkerProvider
+import com.intellij.icons.AllIcons
+import com.intellij.ide.BrowserUtil
+import com.intellij.openapi.editor.markup.GutterIconRenderer
+import com.intellij.psi.PsiElement
+import org.rust.RsBundle
+import org.rust.ide.utils.import.isCore
+import org.rust.ide.utils.import.isStd
+import org.rust.lang.core.psi.RsMetaItem
+import org.rust.lang.core.psi.RsOuterAttr
+import org.rust.lang.core.psi.ext.*
+
+/**
+ * Provide an icon to open the GitHub issue page, e.g. `rust` related.
+ */
+class RsGithubIssueLineMarkerProvider : LineMarkerProvider {
```

Comment:
I don't think we want to provide line marker in this case.
It will add a lot of new UI elements in the editor for all users, although user scope of this feature is quite limited, from my point of view.

Let's better just inject `WebReference` to issue number literal to open browser on `Ctrl + Click` on issue number in the editor.

See `org.rust.lang.core.resolve.ref.RsLiteralWebReferenceProvider` as an example, but your case is much simpler because you don't need to look for url inside literal